Dockerfile

```diff
@@ -27,10 +27,41 @@ WORKDIR /src
 
 FROM gobase AS gotestsum
 ARG GOTESTSUM_VERSION
-ENV GOFLAGS=
-RUN --mount=target=/root/.cache,type=cache \
-  GOBIN=/out/ go install "gotest.tools/gotestsum@${GOTESTSUM_VERSION}" && \
-  /out/gotestsum --version
+ENV GOFLAGS=""
+RUN --mount=target=/root/.cache,type=cache <<EOT
+  set -ex
+  xx-go install "gotest.tools/gotestsum@${GOTESTSUM_VERSION}"
```

Comment:
Why is this using `xx-go` to install. Iiuc these tools should always be for native arch. And I think they are atm as I don't see `ARG TARGETPLATFORM` defined anywhere.